.babelrc.js

```diff
@@ -0,0 +1,29 @@
+const useHMR = !!global.HMR // HotModuleReplacement
+const config = {
+  presets: [
+    '@babel/preset-env',
+    '@babel/preset-react',
+    '@babel/preset-flow'
+  ],
+  plugins: [
+    'babel-plugin-styled-components',
```

Comment:
Do we need all of these plugins?

Reply:
yes, as long as we don't have an integreat babel-preset